server/handlers.go

```diff
@@ -56,10 +59,8 @@ func execTemplate(w http.ResponseWriter, templatePath string, data Data) {
 func readSsidsFile() ([]string, error) {
 	f, err := os.Open(SsidsFile)
 	if err != nil {
-
 		//TODO TRACE
 		log.Printf("ERROR:%v", err)
```

Comment:
ouch!!, please remove that trace. It is a testing one that I forgot to remove in some of my previous landings